src/content/blog/maintainer-job-description.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Recruiting DDEV Open Source Project Maintainers"
+pubDate: 2023-05-19
+summary: The DDEV community is seeking paid maintainers to boost the project's sustainability and growth.
+author: Randy Fay
+featureImage:
+    src: /public/img/blog/2023/05/diverse_group_of_people_sitting_at_a_table_working.png
+    alt: Image of a diverse group of maintainers sitting at a table maintaining DDEV
+categories:
+  - Announcements
```

Comment:
It seems a little strange to me that this is an announcement with a title you’d expect for an announcement, but it doesn’t actually announce anything and leaps straight into a job description—not in a careers section but in the blog.

Reply:
Suggestions on how to categorize it? I initially started with something random, and that failed right away, didn't see first off how to add new category.

Reply:
I [hardcoded categories into the content definition](https://github.com/ddev/ddev.com-front-end/blob/main/src/content/config.ts#L5) with the intention of keeping them consistent, which may be too extreme.

But I wouldn’t recategorize it, I’d write the post as an announcement by giving it a proper introduction.

If it’s worth making a category for job listings, and those listings won’t stay up perpetually (like blog posts presumably should), it’d make more sense to make a separate hiring or careers page instead. Then use the blog to _promote_ new positions if/when necessary instead of being their canonical home.